__tests__/search.test.js

```diff
@@ -38,7 +38,8 @@ describe('Site search', () => {
     const resultsArray = await page.evaluate(
       () => [...document.querySelectorAll('.app-site-search__option')].map(elem => elem.innerText.toLowerCase())
     )
-    expect(resultsArray.every(item => item.startsWith('d'))).toBeTruthy()
+
+    expect(resultsArray.every(item => item.includes('d'))).toBeTruthy()
```

Comment:
Why does this test need to change, I think now this could pass and not meet the original intent for this test.

Reply:
I guess what we want to test for now is:

the results should return with d, or the aliases should start with d

The current amended expectation will return true when any letter is d which feels a bit too loose?

Reply:
aliases don't need to start with d, they just need to contain d

Reply:
Maybe let's pair this test? Think Im doing a crap job at explaining myself :P